RELEASE/scripts/autoscend/quests/level_13.ash

```diff
@@ -991,7 +991,10 @@ boolean L13_towerNSTower()
 			uneffect($effect[Jalape&ntilde;o Saucesphere]);
 			uneffect($effect[Mayeaugh]);
 			uneffect($effect[Spiky Shell]);
-			handleFamiliar($familiar[none]);
+			if (!is100FamRun()) {
+				set_property("auto_disableFamiliarChanging", true);
+				use_familiar($familiar[none]);
```

Comment:
will cause mafia to error in paths which do not allow familiar

Reply:
change to use `canChangeFamiliar()`